tests/api.c

```diff
@@ -5645,6 +5649,306 @@ static void test_wolfSSL_dtls_export(void)
 #endif
 }
 
+
+#if defined(WOLFSSL_SESSION_EXPORT) && !defined(HAVE_IDEA)
```

Comment:
Why is this not supported with `HAVE_IDEA`? The `--enable-all` option turns on IDEA, so worried this new TLS export/import won't be tested...

Reply:
This was because of the macro guard in BulkCipherAlgorithm enum in wolfssl/ssl.h throwing off the correct cipher to use. When looking for wolfssl_aes bulk cipher algorithm type the canned test case did not include the extra enum value. Will see about resolving the limitation of both sides having the same build of IDEA algo on or not.